README.md

```diff
@@ -48,84 +59,126 @@ The script will generate solar forecasts at the specified intervals between the
+3. **Temperature at 2 meters (°C)**, or t : Air temperature measure at 2 meters above the ground. Can affect the efficiency of PV systems.
+4. **Precipiration (mm)**, or prate : Precipitation (rain, snow, sleet, etc.). Helps to predict cloud cover and potentiel reductions in solar irradiance.
+5. **Shortwave Radiation (W/m²)**, or dswrf: Solar radiation in the shortwave spectrum reaching the Earth's surface. Measure of the potential solar energy available for PV systems.
 6. **Direct Radiation (W/m²)** or dlwrf: Longwave (infrared) radiation emitted by the Earth back into the atmosphere. **confirm it is correct**
 7. **Cloud Cover low (%)**, or lcc: Percentage of the sky covered by clouds at low altitudes. Impacts the amount of solar radiation reachign the ground, and similarly the PV system.
-8. **Cloud Cover mid (%)**, or mcc : Percentage of the sky covered by clouds at mid altitudes. 
+8. **Cloud Cover mid (%)**, or mcc : Percentage of the sky covered by clouds at mid altitudes.
 9. **Cloud Cover high (%)**, or lcc : Percentage of the sky covered by clouds at high altitude
+   We also use the following features
 
-We also use the following features
 - poa_global: The plane of array irradiance, which is the amount of solar radiation that strikes a solar panel.
 - poa_global_now_is_zero: A boolean variable that is true if the poa_global is zero at the current time. This is used to help the model learn that the PV generation is zero at night.
 - capacity (kw): The capacity of the PV system in kw.
 - The model also has a feature to check if these variables are NaNs or not.
-
-The model also uses the following variables, which are currently all set to nan
+  The model also uses the following variables, which are currently all set to nan
 - recent_power: The mean power over the last 30 minutes
 - h_mean: The mean of the recent pv data over the last 7 days
 - h_median: The median of the recent pv data over the last 7 days
 - h_max: The max of the recent pv data over the last 7 days
 
+**XGBoost**
+
+The second option is an XGBoost model and uses the following Numerical Weather Predictions (NWP) input features achieved from [open-meteo](https://open-meteo.com/) variables and additional information about the time, location and specifics about the panel. The weather features used are listed below.
+
+- Temperature at 2m (ºC)
+- Relative Humidity at 2m (%)
+- Dewpoint at 2m (ºC)
+- Precipitation (rain + snow) (mm)
+- Surface Pressure (hPa)
+- Cloud Cover Total (%)
+- Cloud Cover Low (%)
+- Cloud Cover Mid (%)
+- Cloud Cover High (%)
+- Wind Speed at 10m (km/h)
+- Wind Direction (10m)
+- Is day or Night
+- Direct Solar Radiation (W/m2)
+- Diffusive Solar Radiation DHI (W/m2)
+
+To use this model specify `model="tryolabs"` in `run_forecast(site=site, model="tryolabs", ts=datetime.today()).
+
```

Comment:
It might be good to explain really clearly what NWP data the model was trained on, These NWP in the past can get very confusing, so its good to get it written down

Reply:
I added the description of each variable from the open-meteo website. Is that what you were referring to?